app/main/views/index.py

```diff
@@ -184,6 +184,12 @@ def download_document(service_id, document_id):
 
 def _format_file_expiry_date(expiry_date_as_str: str) -> str:
     file_expiry_date = (parser.parse(expiry_date_as_str)).date()
+
+    # if expiry date passed, even if file is still available, we do not return it to respect data retention period
+    # set by the service
+    if file_expiry_date - date.today() <= timedelta(days=-1):
```

Comment:
Can we simplify this to something like `if file_expiry_date < date.today()`? 🤔 

Also, not sure if it's just how my brain works, but I find the structure of this condition really hard to read. I find it hard to grok what `file_expiry_date - date.today()` is, but much easier to grok what `date.today() - timedelta(days=1)` is. 🤔  I've had to spent more time than I'd expect understanding how this condition evaluates considering it's just saying "is this date in the past". Maybe that's just my Friday brain >_<

Reply:
I like your one, much simpler ✨ . Will change 👍🏼